Procfile

```diff
@@ -1 +1 @@
-worker: node src/spawn_tests.js
+worker: npm run compile && node lib/main.js test
```

Comment:
That's because the lib folder will be deleted if the dyno stops. Since diagnostics runs only once a day I thought it was safer to have the compile script in the procfile